plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -4864,35 +4866,6 @@ public void testSchemaEvolutionWithDereferenceProjections()
         assertUpdate("DROP TABLE evolve_test");
     }
 
-    @Test
-    public void testHighlyNestedData()
```

Comment:
We'll have these tests enabled now for Iceberg & Delta.
Does this warrant moving them to BCT or should we rather duplicate them in Delta as well?


cc @findepi